### DISCUSSION TITLE
Dag Run Reporting/Archiving into flat files (possibly HTML reports)

### DISCUSSION THREAD

**Discussion by BayanAzima** [apache/airflow#38082] [Q&A] [answered]
I'm regularly purging my airflow instance's metadata db and logs to only retain this data for 90 days. Is there an "established way" to archive this data at a dag instance level into flat files before deleting?

I was thinking per dag run I'd like to create a kind of dump package for achieving purposes. A crude dump of logs and metadata db info would probably work fine though it would be nice to have an html report to click around and explore all the info.

**Comment by potiuk** [comment, marked as answer]
`airflow db clean` create archive tables that you can export and delete the archive tables.
